src/pages/home/index.jsx

```diff
@@ -36,6 +50,18 @@ const Home = () => {
             />
           </InputGroup>
         </Col>
+        if({searchInput.length > 2}){
+        <SearchFilter
+          value={searchInput}
+          data={productData}
+          renderResults={(results) => (
+            <Row className="justify-content-center">
+              {results.map((item, index) => (
+                <ProductCard data={item} key={index} />
```

Comment:
don't use index , use id from json . Using index is not a good practice

Reply:
okay!

Reply:
issue has been fixed